emukit/benchmarking/loop_benchmarking/metrics.py

```diff
@@ -81,3 +84,33 @@ def reset(self) -> None:
         Resets the start time
         """
         self.start_time = time()
+
+
+class CumulativeCostMetric(Metric):
+    """
+    The result is stored in the "metrics" dictionary in the loop state with the key "cumulative_costs"
```

Comment:
Can you extend that a bit, with a description of what is understood as "cumulative cost"?